src/main/webapp/scripts/map.js

```diff
@@ -96,15 +96,23 @@ function initMap() {
   map.addListener("click", e => {
     placeMarkerAndPanTo(e.latLng, map);
   });
-  
-  /* Add Hard-Coded Markers */
-  addMarker(map);
+
+  /* Filter out the recommendations by category if a category parameter is passed */
+  /* this is the case when a category button is clicked */
+  if(category) {
```

Comment:
```suggestion
  if (category) {
```
[4.6.2 Horizontal whitespace](https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace)